graph/push.go

```diff
@@ -25,6 +24,14 @@ import (
 
 var ErrV2RegistryUnavailable = errors.New("error v2 registry unavailable")
 
+type ImagePushConfig struct {
```

Comment:
Ah, ok, I didn't realize that.
If it's only used by graph that's _probably_ fine.